Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/ByAsset.tsx

```diff
@@ -363,6 +363,16 @@ const ByAsset: Application.Types.iByComponent = (props) => {
                                 RowStyle={{ width: '10%' }}
                             > Substations
                             </ReactTable.Column>
+                        <ReactTable.Column<SystemCenter.Types.DetailedAsset>
+                            Key={'Config'}
+                            AllowSort={false}
+                            HeaderStyle={{ width: '8%' }}
+                            RowStyle={{ width: '8%' }}
+                            Content={({ item }) => {
+                                return <NewTabEdit ItemID={item.ID} PageLinkName='Asset' IDLinkName='AssetID' />
```

Comment:
Per previous comment this should be a single button to open in a new tab.
Putting that into it's own component feels overkill. Can be ok if it was a single button